images/macos/toolsets/toolset-10.13.json

```diff
@@ -244,7 +244,8 @@
             "platform" : "darwin",
```

Comment:
Could you please remove PyPy for 10.13 and 10.14?

Reply:
Thank you, removed PyPy 3.7 from 10.13 and 10.14